packages/ecc-client-ga4gh-wes/package.json

```diff
@@ -24,8 +24,8 @@
   "scripts": {
     "analyze": "cem analyze --litelement",
     "build": "node ../../scripts/build.js -p ecc-client-ga4gh-wes-",
-    "dev": "concurrently -k -r \"npm run build -- --watch\" \"wds\"",
-    "clean": "rm -rf dist node_modules custom-elements-manifest.config.js",
+    "dev": "npm run build && concurrently -k -r \"npm run build -- --watch\" \"wds\"",
```

Comment:
there already is build in the command btw but also build wont help us imo, lets say someone runs `npm run dev -w=@elixir-cloud/tes`, `turbo` won't build its deps ie design package, hence imo putting a build here might be disastrous, but as the command was previously that might be better as it build the project on each subs file change.

Reply:
some of these changes must have just been from main, since I pulled from there. I''ll revert them.